aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -221,6 +228,19 @@ export const SignUpExample = () => {
                 }
               />
             </FormField>
+            {credentialError && (
+              <Box
+                direction="row"
```

Comment:
Can we do an `align="center"` here to make the text better aligned w the icon?

<img width="364" alt="Screen Shot 2020-07-29 at 10 37 08 AM" src="https://user-images.githubusercontent.com/12522275/88833948-f33ef100-d187-11ea-84e4-8ce0bfe16a41.png">

Maybe instead of using `strong` we should use `Text` with `weight="bold"` to keep it more grommety?

Reply:
Actually I just looked back at the final designs, and it looks like it's calling for a different icon now:
<img width="485" alt="Screen Shot 2020-07-29 at 10 44 48 AM" src="https://user-images.githubusercontent.com/12522275/88834382-998af680-d188-11ea-885b-0d64425d14a1.png">

Reply:
yeah I wasnt to worried about the alignment with icon since I knew it was going to be switched out